applications/poisson/sine/tests/curvilinear_AMG.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
new test to still cover AMG for scalar-valued equations (constant modes vector is different, code paths are different and is not tested elsewhere)